src/Service/Mailer/Mail/AbstractNotificationMailer.php

```diff
@@ -29,10 +31,11 @@ public function __construct(
         protected ParameterBagInterface $parameterBag,
         protected LoggerInterface $logger,
         protected UrlGeneratorInterface $urlGenerator,
+        private EntityManagerInterface $entityManager,
```

Comment:
Pour éviter de mettre à jour tous les enfants, l'alternative serait de passer par une injectiion au niveau d'un setter 
https://symfony.com/doc/current/service_container/calls.html#:~:text=To%20configure%20the,4%0A5%0A6

Reply:
j'ai essayé, ça ne marche pas

`The service "App\Service\Mailer\Mail\AbstractNotificationMailer" has a dependency on a non-existent service "entityManager".`